scripts/crawl.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable promise/always-return */
+const { crawl } = require('../lib/crawl')
+
+const crawls = [
+  crawl('https://sfgov.org/bac/', [
+    '--include-directories', '/sfc/bac/'
+  ]),
+  crawl('https://sfgov.org/arts/', [
+    // '--reject=.pdf'
+  ]),
+  crawl('https://sfgov.org/dosw/', [
+  ])
```

Comment:
I guess it kinda depends on how many crawls we'd do at a time, as well? If it's like a handful, this could be fine.. that just means we have to limit how many we do per run and do multiple runs... if we want a higher level one, we could also do that which will make sure there's only N maximum request made at any given time, too. Depends on how fancy we wanna get.

Reply:
I think for this type of thing, language isn't going to be too big of a problem... just pick something people are familiar with and something that has enough libs out there that we can just pull from to get new stuff (so Node is totally fine IMO).

Yea, I'd agree. Nothing fancy.. just enough to not make it suck.. especially if we have to do progressive crawling of sites. And "a human can be as an orchestrator" is certainly a reasonable answer... to a point. 😅